stdlib/printexc.mli

```diff
@@ -193,7 +193,7 @@ val backtrace_slots : raw_backtrace -> backtrace_slot array option
     recent function call, raise, or primitive [get_backtrace] call in
     the trace.
 
-    Some possible reasons for returning [None] are as follow:
+    Some possible reasons for returning [None] are as follows:
```

Comment:
British English speakers outnumber American English speakers on the dev team :wink:

My French is terrible, but I wonder if the sentence is simply a literal translation of the French - you could ask @gasche, as he wrote it!